spec/compiler/semantic/tuple_spec.cr

```diff
@@ -283,11 +283,22 @@ describe "Semantic: tuples" do
       require "prelude"
 
       pos = {0, 0}
-      while true
+      301.times do
```

Comment:
Why don't you leave it at `while true`?

Adding two members 301 times would - if there was no error - end up in a tuple of size 604 anyway.